### PR TITLE
Update `rna-seq-expression-summary-stats` module for v11 data

### PR DESCRIPTION
**Pull request by sangeetashukla**
### Purpose/implementation Section
Update the `rna-seq-expression-summary-stats` module to run successfully with v11 data and generate appropriate result files.

#### What scientific question is your analysis addressing?
The v11 data release files are not compatible with the old/existing module, and therefore the module needs to be updated and any errors handled.

#### What was your approach?
- Updated the module to use below input files
`independent-specimens.rnaseqpanel.primary.tsv`
`independent-specimens.rnaseqpanel.primary.eachcohort.tsv`

#### What GitHub issue does your pull request address?
[Issue 413](https://github.com/PediatricOpenTargets/ticket-tracker/issues/413)

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
Please confirm the input files are as expected.

#### Which areas should receive a particularly close look?
The calculation of summary stats for `gene-wise z-scores` generated 134 `NaN`s for the below list of genes, which was replaced with zero.

MIR1244-2 | ENSG00000283498
SNORD96A | ENSG00000272296
RNA5SP122 | ENSG00000207186
MIR1244-4 | ENSG00000283475
MIR1244-3 | ENSG00000283429
MIR4258 | ENSG00000264349
RNA5SP345 | ENSG00000222578
RNU6-1065P | ENSG00000207242
RNU6-88P | ENSG00000272507
MIR126 | ENSG00000199161
MIR1285-1 | ENSG00000221520
RNU6-917P | ENSG00000252103
RNU6-1153P | ENSG00000252569
MIR3178 | ENSG00000266232
RNU6-430P | ENSG00000252887
RNU6-335P | ENSG00000201433
LINC01818 | ENSG00000230645
LINC02220 | ENSG00000248457
MIR141 | ENSG00000207708
RNU6-1208P | ENSG00000238482
MIR3183 | ENSG00000264429
RNU6-704P | ENSG00000201032
RNA5SP101 | ENSG00000252845
SNORD51 | ENSG00000207047,ENSG00000202275
RNU6-877P | ENSG00000206595
MIR5008 | ENSG00000264483
MIR194-1 | ENSG00000207624
snoU2_19 | ENSG00000199231,ENSG00000199894,ENSG00000201592,ENSG00000275146
AC073218.1 | ENSG00000226785,ENSG00000232153
MIR3621 | ENSG00000263697
RNU6-1304P | ENSG00000223306
RNU6-1150P | ENSG00000252606
RNU6-792P | ENSG00000200360
MIR550A3 | ENSG00000212024
MIR342 | ENSG00000199082
RNA5SP79 | ENSG00000202063
RNU6-504P | ENSG00000252377
RNU6-1188P | ENSG00000200665
MIR6716 | ENSG00000284246
MIR8070 | ENSG00000278529
RNU6-861P | ENSG00000252554
SNORD105B | ENSG00000238531
SNORD14 | ENSG00000202479
RNA5SP330 | ENSG00000252905
RNU6-128P | ENSG00000206703
RNU6-537P | ENSG00000253086
RNA5SP86 | ENSG00000212171
RNU6-173P | ENSG00000201085
MIR340 | ENSG00000198995
AC027124.2 | ENSG00000223398,ENSG00000238138
MIR603 | ENSG00000207930
RNU6-1148P | ENSG00000222924
S100A7L2 | ENSG00000197364
RNU6-601P | ENSG00000207270
RNA5SP373 | ENSG00000253080
RNU6-344P | ENSG00000252444
MIR125B1 | ENSG00000207971
MIR6127 | ENSG00000276835
RNU6-615P | ENSG00000251874
RNA5SP36 | ENSG00000222129
RNA5SP436 | ENSG00000251829
RNA5SP456 | ENSG00000200872
RNU6-74P | ENSG00000206962
RNU6-1119P | ENSG00000207077
RNU6ATAC29P | ENSG00000221015
RNU6-64P | ENSG00000252928
MIR125B2 | ENSG00000207863
TBC1D3P3 | ENSG00000235203
RNU6-1133P | ENSG00000222895
RNA5SP470 | ENSG00000252272
RNU6-580P | ENSG00000202029
RNU6-662P | ENSG00000206980
MIR498 | ENSG00000207869
RNU6-296P | ENSG00000207338
RNA5SP485 | ENSG00000201923
SNORD115-41 | ENSG00000200478
RNU6-1292P | ENSG00000206583
GSTA11P | ENSG00000230375
NUTF2P8 | ENSG00000270654
SNORD115-8 | ENSG00000200726
MIR4306 | ENSG00000263615
CYP2C59P | ENSG00000228460
RNU6-189P | ENSG00000251929
RNA5SP346 | ENSG00000200411
RNU6-1180P | ENSG00000201805
RNA5SP110 | ENSG00000199508
RNU6-461P | ENSG00000201065
MIR802 | ENSG00000211590
SNORD115-15 | ENSG00000201679
RNU6-1246P | ENSG00000212366
RNU6-926P | ENSG00000206802
RNU6-656P | ENSG00000252594
SNORD115-38 | ENSG00000201907
SNORD115-10 | ENSG00000201943
RNA5SP426 | ENSG00000199771
SNORD115-39 | ENSG00000200564
RNU6-1124P | ENSG00000206896
RNA5SP64 | ENSG00000207341
MIR424 | ENSG00000284231
MIR1302-4 | ENSG00000221628
AP001506.1 | ENSG00000276738
RNA5SP134 | ENSG00000252301
RNU6-1286P | ENSG00000206598
RNU6-154P | ENSG00000207033
MIR505 | ENSG00000207633
RNU6-829P | ENSG00000200570
RNU6-797P | ENSG00000199971
MIR30B | ENSG00000207582
RNU6-183P | ENSG00000200331
RNU6-203P | ENSG00000206765
RNU6-200P | ENSG00000201613
MTCO1P57 | ENSG00000232975
RNU6-224P | ENSG00000206782
MIR5100 | ENSG00000263795
RNU6-185P | ENSG00000207150
RNA5SP337 | ENSG00000252816
RNU1-15P | ENSG00000195024
RNU6-281P | ENSG00000206939
RNA5SP56 | ENSG00000222209
AC107399.1 | ENSG00000223605
RNA5SP171 | ENSG00000212373
MIR203A | ENSG00000207568
KRTAP5-14P | ENSG00000249387
TSPY16P | ENSG00000226223
AC007967.2 | ENSG00000228207
RNU6-1012P | ENSG00000252415
RNA5SP510 | ENSG00000201567
MIR512-2 | ENSG00000207644
TRIM60P8Y | ENSG00000232764
ZNF736P12Y | ENSG00000232003
TRAPPC2P10 | ENSG00000228465
TRIM60P10Y | ENSG00000232976
MIR3655 | ENSG00000284325
USP9YP9 | ENSG00000232614

#### Is there anything that you want to discuss further?
Note: Since the input is rnaseq.panel data, the samples have not filtered out DGD cohort. Let me know in case we still want to implement filtering.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### What types of results are included (e.g., table, figure)?
Tables

#### What is your summary of the results?
`results\long_n_tpm_mean_sd_quantile_gene_wise_zscore.jsonl.gz`
`results\long_n_tpm_mean_sd_quantile_group_wise_zscore.jsonl.gz`
`results\long_n_tpm_mean_sd_quantile_gene_wise_zscore.tsv.gz`
`results\long_n_tpm_mean_sd_quantile_group_wise_zscore.tsv.gz`
`results\cancer_group_each_cohort_sample_metadata.tsv`
`results\cancer_group_all_cohorts_sample_metadata.tsv`

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by jharenza** [comment]
@sangeetashukla and @logstar are we still using this module? I thought this one was only being generated via the api now, but maybe I am missing something.

**Comment by sangeetashukla** [comment]
> @sangeetashukla and @logstar are we still using this module? I thought this one was only being generated via the api now, but maybe I am missing something.

@jharenza @logstar Then I can remove it from CI once you can confirm it is generated via API.

**Comment by ewafula** [comment]
@atlas4213 and @rjcorb,  would you mind prioritizing reviewing this PR so @sangeetashukla can complete it, and then we can merge?  We need to deliver the `tpm table` to FNL for the internal QC on the MTP portal. 
cc @chinwallaa

**Comment by logstar** [comment]
> @sangeetashukla and @logstar are we still using this module? I thought this one was only being generated via the api now, but maybe I am missing something.

@jharenza - Thank you for checking.

I think we will still be using this module to generate TPM JSONL files to share with the FNL team.

It is correct that the API generates TPM boxplots and tables, but the TPM JSONL files are still used by the MTP navigation page.

Let me know if you have questions.

**Review by atlas4213 — APPROVED** [review]
I looked through the produced files and saw no NA's so it looks good to me

File: .github/workflows/continuous_integration.yml [hunk 1]
```diff
@@ -136,6 +136,9 @@ jobs:
           - name: Update table annotation data
             entrypoint: long-format-table-utils/run-update-long-format-table-utils.sh
 
+          - name: RNA-Seq Expression Summary stats
+            entrypoint: rna-seq-expression-summary-stats/run-rna-seq-expression-summary-stats.sh
+
```
Comment (ewafula) [inline .github/workflows/continuous_integration.yml:141]: @sangeetashukla , Need to move this under `# MTP-specific modules` below. 

I am not certain if this has to be a separate PR that needs to be merged first for successful GA testing for the `rna-seq-expression-summary-stats` module

Cc @jharenza

**Comment by ewafula** [comment]
@sangeetashukla,  as we discussed this morning with @logstar @chinwallaa,  setting zero to where the z-score values are `NANs` instead of blanks is okay. I have looked through it as @rjcorb  did; it results from the computed means and standard deviations being zero and not missing data. 

Two remaining issues before PR can be merged:

1). GA error ( it didn't pass):
``` 
docker: Error response from daemon: failed to create shim: OCI runtime create failed: runc create failed: unable to start container process: exec: "analyses/rna-seq-expression-summary-stats/run-rna-seq-expression-summary-stats.sh": permission denied: unknown. 
```
2). Update README with correct input files used in the module

**Review by ewafula — APPROVED** [review]
